cmd/k8s-operator/operator_test.go

```diff
@@ -968,7 +968,7 @@ func expectedEgressSTS(stsName, secretName, tailnetTargetIP, hostname, priorityC
 					InitContainers: []corev1.Container{
 						{
 							Name:    "sysctler",
-							Image:   "busybox",
+							Image:   "tailscale/tailscale",
```

Comment:
@maisem should the image here be specified as `ghcr.io/tailscale/tailscale`?

Reply:
we publish to both, so both are valid.

For the context of these tests, their setup sets `proxyImage=tailscale/tailscale` so this is correct in this instance.

Reply:
OK